src/System.Net.Primitives/tests/FunctionalTests/CookieCollectionTest.cs

```diff
@@ -156,7 +156,17 @@ public static void Enumerator_Version_Invalid()
             enumerator.MoveNext();
 
             cc.Add(new Cookie("name5", "value"));
-            Assert.NotNull(enumerator.Current);
+
+            object current = null;
+            var exception = Record.Exception(() => current = enumerator.Current);
+
+            // on desktop, enumerator.Current throw an exception because the collection has been modified after 
```

Comment:
nit: Please capitalize sentences in comments similar to the existing coding style. this comment applies with all of the changes in this PR.